src/models/account.cpp

```diff
@@ -507,6 +526,185 @@ int Account::importFromCSV(const std::string& path)
+                    {
+                        try
+                        {
+                            transaction->setAmount(boost::multiprecision::cpp_dec_float_50(content));
+                        }
+                        catch (...)
+                        {
+                            skipTransaction = true;
+                            continue;
+                        }
+                    }
+                    // Date
+                    if (tag == "DTPOSTED")
+                    {
+                        try
+                        {
+                            transaction->setDate(boost::gregorian::from_undelimited_string(content));
+                        }
+                        catch (...)
+                        {
+                            skipTransaction = true;
+                            continue;
+                        }
+                    }
+                }
+            }
+        }
+    }
+
+    return imported;
+}
+
+int Account::importFromQIF(const std::string& path)
+{
+    int imported{ 0 };
+    unsigned int nextId{ getNextAvailableTransactionId() };
+    std::ifstream file{ path };
+    Transaction *transaction{ new Transaction(nextId) };
+    bool skipTransaction{ false };
+    if (file.is_open())
+    {
+        std::string line;
+        while (getline(file, line))
+        {
+            const long unsigned int found = line.find('\r');
+            if (found != std::string::npos)
+            {
+                line.erase(found);
+            }
+            if (line.length() == 0)
+            {
+                continue;
+            }
+            // Add the previous transaction
+            if (line[0] == '^')
+            {
+                if (!skipTransaction)
+                {
+                    addTransaction(*transaction);
+                    imported++;
+                }
+                delete transaction;
+                skipTransaction = false;
+                nextId++;
+                transaction = new Transaction{ nextId };
+            }
+            // Date
+            if (line[0] == 'D')
+            {
+                try
+                {
+                    const int year{ atoi(line.substr(7, 4).c_str()) };
+                    const int month{ atoi(line.substr(4, 2).c_str()) };
+                    const int day{ atoi(line.substr(1, 2).c_str()) };
+                    transaction->setDate(boost::gregorian::date(year, month, day));
+                }
+                catch (...)
+                {
+                    skipTransaction = true;
+                    continue;
+                }
+            }
+            // Amount and Type
+            if (line[0] == 'T')
+            {
+                try
+                {
+                    transaction->setType(line[1] == '-' ? TransactionType::Expense : TransactionType::Income);
+                    transaction->setAmount(boost::multiprecision::cpp_dec_float_50(line.substr(2)));
```

Comment:
Use `{}-initialization`

```cpp
transaction->setAmount({ line.substr(2) });
```
Removes the need for specifying `boost::multiprecision::cpp_dec_float_50`

Reply:
This unfortunately does not work:
```
error: cannot convert '<brace-enclosed initializer list>' to 'boost::multiprecision::cpp_dec_float_50' {aka 'boost::multiprecision::number<boost::multiprecision::backends::cpp_dec_float<50> >'}
```

Reply:
Oh ok, then you can leave it as `transaction->setAmount(boost::multiprecision::cpp_dec_float_50(line.substr(2)));`